omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml

```diff
@@ -0,0 +1,846 @@
+                                                    <label>Clinical Team Leader</label>
+                                                    <encounterProvider role="Provider" style="autocomplete"/>
+                                                </div>
+                                            </div>
+                                            <div class="row">
+                                                <div class="col-md-3">
+                                                    <lookup expression="fn.patientAgeInDays()"
+                                                            class="patientAgeInDays"/>
+                                                    <lookup expression="fn.patientAgeInMonths()"
+                                                            class="patientAgeInMonths"/>
+                                                    <label>Date of NVP Start</label>
+                                                    <obs conceptId="99771" id="nvp-date"/>
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <label>Date of CTX Start</label>
+                                                    <obs conceptId="99773" id="ctx-date"/>
+                                                </div>
+                                                <span class="container-padding"></span>
+                                                <label>Birth Defect?</label>
+                                                <div class="col-md-2">
+                                                    <obs conceptId="165402" answerConceptIds="1065,1066"
+                                                         answerLabels="Yes,No" style="radio" class="horizontal"/>
+                                                </div>
+                                                <label>Neurotube Defect?</label>
+                                                <div class="col-md-2">
+                                                    <obs conceptId="165403" answerConceptIds="1065,90002"
+                                                         answerLabels="Yes,Others" style="radio" class="horizontal"/>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="card">
+                                        <div class="card-header">
+                                            Mother/Care Giver Follow Up Information
+                                        </div>
+                                        <div class="card-body">
+                                            <div class="row">
+                                                <div class="col-md-3">
+                                                    <label>Mother`s Name</label>
```

Comment:
@semujju this has a strange apostrophe character

Reply:
This character has been removed and a correct one used